aptos-move/aptos-release-builder/data/enable-loader-v2.yaml

```diff
@@ -0,0 +1,13 @@
+---
+remote_endpoint: ~
+name: "v1.24.0"
```

Comment:
@sherry-x what do we use for name for feature flag enablements? I don't think we use release numbers, but what the feature is?

Reply:
Renamed to "v1.24-enable-v2-loader"